applications/matlab-batch/app.json

```diff
@@ -0,0 +1,101 @@
+{
+  "id": "matlab-batch",
+  "version": "23.2",
+  "description": "MATLAB Batch is a non-interactive application to run *.m scipts.  Submit a help ticket to get your license configured or approved.",
+  "owner": "${apiUserId}",
+  "runtime": "ZIP",
+  "runtimeVersion": null,
+  "runtimeOptions": null,
+  "containerImage": "tapis://cloud.data/corral/tacc/aci/CEP/applications/v3/matlab-batch/2023b/matlab-batch.zip",
+  "jobType": "BATCH",
+  "maxJobs": -1,
+  "maxJobsPerUser": -1,
+  "strictFileInputs": true,
+  "jobAttributes": {
+    "description": "",
+    "dynamicExecSystem": false,
+    "execSystemConstraints": null,
+    "execSystemId": "stampede3",
+    "execSystemExecDir": "${JobWorkingDir}",
+    "execSystemInputDir": "${JobWorkingDir}",
+    "execSystemOutputDir": "${JobWorkingDir}/output",
+    "execSystemLogicalQueue": "skx-dev",
+    "archiveSystemId": "cloud.data",
+    "archiveSystemDir": "HOST_EVAL($HOME)/tapis-jobs-archive/${JobCreateDate}/${JobName}-${JobUUID}",
+    "archiveOnAppError": true,
+    "isMpi": false,
+    "mpiCmd": null,
+    "cmdPrefix": null,
+    "parameterSet": {
+      "appArgs": [],
+      "containerArgs": [],
+      "schedulerOptions": [
+        {
+          "name": "TACC Scheduler Profile",
+          "description": "Scheduler profile for VisIt",
+          "inputMode": "FIXED",
+          "arg": "--tapis-profile matlab-2023b",
+          "notes": {
+            "isHidden": true
+          }
+        },
+        {
+          "name": "Slurm job name",
+          "description": "Set the slurm job name to be identical to the Tapis job name.",
+          "inputMode": "FIXED",
+          "arg": "--job-name ${JobName}",
+          "notes": {
+            "isHidden": true
+          }
+        }
+      ],
+      "envVariables": [
+        {
+          "key": "matlabScriptName",
+          "value": "",
+          "description": "Provide the filename of the MATLAB script to execute, relative to the path of the Working Directory. This file should reside in the Working Directory specified. Example: 'mymatlabtest.m'.",
+          "inputMode": "REQUIRED",
+          "notes": {
+            "label": "MATLAB Script Filename"
+          }
+        }
+      ],
+      "archiveFilter": {
+        "includes": [
+          "workingDirectory"
```

Comment:
this will not copy workingDirectory to archive folder. 
If you want file inputs to be copied, then, my suggestion is to do these 3 changes:
1) Set input and output folder to be same, example: https://github.com/TACC/WMA-Tapis-Templates/blob/main/applications/mpm/app.json#L21-L22
2) set includes to be empty array.
3) set excludes to exclude zip file "matlab-batch.zip"